### DISCUSSION TITLE
Pinned repo display bug.

### DISCUSSION THREAD

**Discussion by woolson** [community/community#4129] [Mobile] [answered]
### General information

- Platform: iOS@14.6
- Github Mobile Ver: 1.17.0

### Description

An pinned repo stick on right side.

### Screenshot

<img src="https://user-images.githubusercontent.com/27878293/120979340-1672fd80-c7a8-11eb-9618-66a4cbffb71e.PNG" width="350"/>

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the report @woolson! I have been able to reproduce the bug myself, so I will create an issue to track it on our side. I hope our engineers can fix it soon. Please, keep the feedback coming!

**Reply by woolson** [reply]
Ok
